packages/components/src/unit-control/index.tsx

```diff
@@ -259,7 +259,6 @@ function UnforwardedUnitControl(
 	return (
 		<Root className="components-unit-control-wrapper" style={ style }>
 			<ValueInput
-				aria-label={ label }
```

Comment:
⚠️ Runtime change: This `label` variable is a `ReactNode`, but `aria-label` only accepts strings. I went ahead and removed this redundant `aria-label`, because the same value is added on line 270 via `label` prop.